kobuki_keyop/include/kobuki_keyop/keyop.hpp

```diff
@@ -38,25 +38,32 @@
  ** Ifdefs
  *****************************************************************************/
 
-#ifndef KEYOP_CORE_NODE_HPP_
-#define KEYOP_CORE_NODE_HPP_
+#ifndef KEYOP_HPP_
```

Comment:
Probably should guard these with a unique name here - I usually use the package name. `KOBUKI_KEYOP_HPP_` should do the trick.

Reply:
Good call, fixed in 1e2e665.